client/jetpack-connect/authorize.js

```diff
@@ -1280,10 +1270,7 @@ export class JetpackAuthorize extends Component {
 			/>
 		);
 
-		if (
-			config.isEnabled( 'jetpack/onboarding-user-connection-redesign' ) &&
-			this.isFromJetpackOnboarding()
-		) {
```

Comment:
@grzegorz-cp One thing that is suspicious about this customization is that it skips rendering the legal disclaimer (you agree to share details with WordPress.com...). Was that really the intent? Maybe the disclaimer was moved to another place in the flow?

Reply:
Thanks, I was able to find it in the Jetpack repo, in the `my-jetpack` package :+1: